types/office-js-preview/index.d.ts

```diff
@@ -22590,8 +23225,7 @@ declare namespace Excel {
         getEntireRow(): Excel.Range;
         /**
          * Renders the range as a base64-encoded png image.
-         *
-         * **Important**: This API is currently unsupported in Excel for Mac. Visit [OfficeDev/office-js Issue #235](https://github.com/OfficeDev/office-js/issues/235) for the current status.
+                    **Important**: This API is currently unsupported in Excel for Mac. Visit [OfficeDev/office-js Issue #235](https://github.com/OfficeDev/office-js/issues/235) for the current status.
```

Comment:
Does this need to use `{@link...}` notation ([current behavior in docs](https://docs.microsoft.com/en-us/javascript/api/excel/excel.range?view=excel-js-preview#getimage--))? Here and other places. (Same issue here: https://docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.range?view=office-scripts#getimage--)
```suggestion
                    *
                    * **Important**: This API is currently unsupported in Excel on Mac. Visit {@link https://github.com/OfficeDev/office-js/issues/235 | OfficeDev/office-js Issue #235} for the current status.
```